extensions/PhabBugz/lib/Util.pm

```diff
@@ -36,11 +37,18 @@ our @EXPORT = qw(
 
 sub get_revision_by_id {
     my $id = shift;
+    my @result_data = get_revisions_by_ids(qw($id));
+
+    return $result_data[0];
+}
+
+sub get_revisions_by_ids {
+    my ($ids) = @_;
 
     my $data = {
         queryKey => 'all',
         constraints => {
-            ids => [ int($id) ]
+            ids => join(',', $ids)
```

Comment:
array reference i meant to say. join() returns a list where is a [ ] is a reference to a list which is what JSON wants.